<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>VersionTrackerDialogUI</class>
 <widget class="QDialog" name="VersionTrackerDialogUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Track File(s)</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>3</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="mName" />
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="mPrefix" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="textLabel3" >
       <property name="text" >
        <string>Filename Prefix:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="textLabel2" >
       <property name="text" >
        <string>Name:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="textLabel4" >
     <property name="text" >
      <string>Tracked Files:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="mFileList" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>51</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOK" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>mOK</sender>
   <signal>clicked()</signal>
   <receiver>VersionTrackerDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>230</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel" >
     <x>135</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCancel</sender>
   <signal>clicked()</signal>
   <receiver>VersionTrackerDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>335</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel" >
     <x>336</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
